lib/tasks/transform-files.js

```diff
@@ -23,9 +25,21 @@ export default async function modifyFiles(options) {
   if (options.ts) {
     if (existsSync('types/index.d.ts')) {
       await transformWithReplace('types/index.d.ts', transformTypesIndex);
-    }
+    } else {
+      const source = await readFile('tsconfig.json', 'utf8');
+      const before = parse(source);
+      const beforeValue = get(before, 'compilerOptions.types') ?? [];
+      const newValue = [
```

Comment:
so this isn't idempotent any more. We should only add things if they don't already exist 👍

Reply:
The Set would filter duplication, but I guess this would potentially reorder things in the file. 

I've pushed a new commit with a check.